en/mkdocs.yml

```diff
@@ -52,34 +52,38 @@ strict: true
 # Navigation
 nav:
     - Home: index.md
-    - Services:
-          - Service Concepts: services/service-concepts.md
-          - Create Your First Service: services/create-your-first-service.md
-    - APIs:
-          - API Concepts: apis/api-concepts.md
-          - Create and Publish Your First API: apis/create-and-publish-your-first-api.md
-    - Integrations:
-          - Integration Concepts: integrations/integration-concepts.md
-          - Build Your First Integration: integrations/build-your-first-integration.md
-    - Remote Apps:
-          - Connect Remote Applications: remote-apps/connect-remote-applications.md
-    - Observability:
-          - Observability Overview: observability/observability-overview.md
-          - Perform Root Cause Analysis: observability/perform-root-cause-analysis.md
-          - Performance Analysis: observability/analyze-performance.md
-    - Insights:
+    - Concepts:
+          - Concepts: concepts/concepts.md
+    - Quick Start Guide:
+          - Quick Start Guide: qsg/quick-start-guide.md
+    - Develop:
+          - Create API: develop/create-api.md
+          - Create Integration:
+                - Create Integration: develop/create-integration.md
+                - Ballerina Expression Syntax: references/ballerina-expression-syntax.md
+                - Data Mapping: references/data-mapping.md
+                - Connectors : references/connectors.md
+    - Deploy:
+          - Deploy APIs: deploy/deploy-apis.md
+          - CI/CD: deploy/ci-cd.md
+    - Manage:
```

Comment:
To this section let's just add a general page.

Reply:
Done